README.md

```diff
@@ -1,207 +1,225 @@
+| `signalfx.span.tags` | `SIGNALFX_SPAN_TAGS` | `null` | Comma-separated list of tags included in every reported span. For example, `"key1:val1,key2:val2"`. |
 | `signalfx.db.statement.max.length` | `SIGNALFX_DB_STATEMENT_MAX_LENGTH` | `1024` | The maximum number of characters written for the OpenTracing `db.statement` tag. |
 | `signalfx.recorded.value.max.length` | `SIGNALFX_RECORDED_VALUE_MAX_LENGTH` | `12288` | The maximum number of characters for any Zipkin-encoded tagged or logged value. |
-| `signalfx.trace.annotated.method.blacklist` | `SIGNALFX_TRACE_ANNOTATED_METHOD_BLACKLIST` | `null` | Prevent `@Trace` annotation functionality for the target method string of format `package.OuterClass[methodOne,methodTwo];other.package.OuterClass$InnerClass[*];` (`;` is required and `*` for all methods in class). |
-| `signalfx.max.spans.per.trace` | `SIGNALFX_MAX_SPANS_PER_TRACE` | `0 (no limit)` | Drop (don't deliver) traces with more spans than this. Intended to prevent runaway traces from flooding upstream systems. |
-| `signalfx.max.continuation.depth` | `SIGNALFX_MAX_CONTINUATION_DEPTH` | `100` | Stop propagating asynchronous context at this recursive depth. Intended to prevent runaway traces from leaking memory. |
+| `signalfx.trace.annotated.method.blacklist` | `SIGNALFX_TRACE_ANNOTATED_METHOD_BLACKLIST` | `null` | Prevents `@Trace` annotation functionality for the target method string of format `package.OuterClass[methodOne,methodTwo];other.package.OuterClass$InnerClass[*];`. All methods in class need to include `*` and `;`. |
+| `signalfx.max.spans.per.trace` | `SIGNALFX_MAX_SPANS_PER_TRACE` | `0 (no limit)` | Drops traces with more spans than this. Intended to prevent runaway traces from flooding upstream systems. |
+| `signalfx.max.continuation.depth` | `SIGNALFX_MAX_CONTINUATION_DEPTH` | `100` | Stops propagating asynchronous context at this recursive depth. Intended to prevent runaway traces from leaking memory. |
 
-**Note: System property values take priority over corresponding environment
-variables.**
+### Steps
 
-**Note: To enable debug logging for troubleshooting assistance, please set the
-`datadog.slf4j.simpleLogger.defaultLogLevel=debug` property at runtime.  These logs
-are extremely verbose.**
+Follow these steps to configure the agent to send traces for `your_app` to a
+Smart Agent available on `localhost`. To send traces to a remote Smart
+Agent, instead specify the `signalfx.agent.host` system property or
+`SIGNALFX_AGENT_HOST` environment variable before you include the Java Agent in your
+application.
 
-```bash
-$ export SIGNALFX_SERVICE_NAME=my-app
-$ export SIGNALFX_SPAN_TAGS="MyTag:MyValue"
-$ java -javaagent:signalfx-tracing.jar <rest of command-line arguments...>
-```
+1. Download the latest version of the agent from the
+[releases](https://github.com/signalfx/signalfx-java-tracing/releases) page.
+2. Set the required environment variables or system properties for your
```

Comment:
properties are set w/ the java invocation or in properties files.  No real reason to make this a separate step since the env vars are for the java process.

Reply:
will the provided steps work just as well with step 2 removed?

Reply:
yes, with the default service name.  you can also have a single instruction:
`SIGNALFX_SERVICE_NAME="your_app" java -javaagent...`

Reply:
going to leave in the step so users can name the service before sending data in.